test/e2e/scheduler_predicates.go

```diff
@@ -147,6 +147,9 @@ var _ = Describe("SchedulerPredicates", func() {
 		framework.beforeEach()
 		c = framework.Client
 		ns = framework.Namespace.Name
+		var err error
+		nodeList, err = c.Nodes().List(labels.Everything(), fields.Everything())
```

Comment:
Otherwise I'll get segfaults in tests that use this - it was accidentally broken by me and Quinton today, so be better fix this quickly:)